app/src/main/java/com/android/solvit/shared/ui/map/MapScreen.kt

```diff
@@ -121,6 +170,11 @@ fun MapContent(
       }
 }
 
+/**
+ * Composable that displays a marker on the map.
+ *
+ * @param markerData The data to display on the marker.
+ */
 @Composable
 fun MapMarker(markerData: MarkerData) {
   MarkerComposable(
```

Comment:
You could animate the selected markers with a code of this style
```
val selected = remember { mutableStateOf(false) }
MarkerComposable(
    keys = arrayOf(markerData.tag),
    state = MarkerState(position = markerData.location),
    onClick = {
        selected.value = !selected.value
        markerData.onClick()
        true
    }
) {
    OutlinedCard(
        modifier = Modifier
            .width(80.dp)
            .height(100.dp)
            .testTag(markerData.tag)
            .scale(animateFloatAsState(if (selected.value) 1.1f else 1f).value),
        shape = RoundedCornerShape(8.dp),
        elevation = CardDefaults.outlinedCardElevation(
            defaultElevation = if (selected.value) 8.dp else 4.dp
        )
    ) {
        // ... existing content ...
    }
}
```

Reply:
since we navigate onClick this is not useful